migrations/1_initial_migration.js

```diff
@@ -2,7 +2,16 @@ const path = require('path')
 const fs = require('fs').promises
 const WorkflowCompile = require('@truffle/workflow-compile')
 
+const WEI_TO_DEPOSIT = web3.utils.toWei("10", "ether")
+const DAI_TO_CREATE  = web3.utils.toWei("80", "ether")
+
+const FIXED_FEED_PRICE = web3.utils.toWei("190", "gether") // Unit = 1e27
+const DEBT_CEILING     = web3.utils.toWei("10000000000000000000000", "gether") // Unit = 1e45
```

Comment:
My personal preference would be to use native `bigint`s throughout the code and then convert to a base10 number at the end.  In other projects I would just create a couple helper methods like `toEth(attoeth:bigint)` and `toAttoeth(eth:bigint)` to do most of the lifting.  `bigint` to base10 string is as easy as `someBigintValue.toString(10)`.

Reply:
Regardless of how we actually store data, we should put units on these variable names for clarity I think.